bgcval2/analysis_compare.py

```diff
@@ -1812,7 +1813,7 @@ def load_yml_and_run(compare_yml, config_user):
     timeranges = details['timeranges']
     suites = details['suites']
     auto_download = details['auto_download']
-
+    strictFileCheck = details.get('strictFileCheck', True)
```

Comment:
This is where strictFileCheck gets set. the details is a dict loaded directly from the input_yml.

Reply:
Added a note into the README.

Reply:
so is this an actual file integrity check or a file existence check?

Reply:
or this is a shimmy for a future implementation of a file integrity check?

Reply:
> Added a note into the README.

gotcha! But I still want to see eg `if strictFileCheck: <some check on the existence of files>` ie as far as I can see there is no functionality behind this flag

Reply:
It existed already, so it's not in the PR.

Reply:
can you point me to the code pls?